src/function.js

```diff
@@ -0,0 +1,117 @@
+import './style.css';
+/* eslint-disable */
+const form = document.getElementById('#form');
+const inputcontainer = document.getElementById('inputcontainer');
+const userInput = document.getElementById('userInput');
+const userTasks = document.getElementById('userTasks');
+
+inputcontainer.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    addToDo(userInput.value);
+  }
+});
+
+let data = [];
+
+const addToDo = (item) => {
+  if (item !== '') {
+    const todo = {
+      description: String,
+      completed: false,
+      index: Number
+    };
+    todo.description = item;
+    todo.index = data.length;
```

Comment:
- [ ] Kindly notice that the property `index` is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) so the index should be assigned like this.

```javascript
todo.index = data.length + 1;
```